src/gui/folderstatusdelegate.cpp

```diff
@@ -59,7 +59,7 @@ QSize FolderStatusDelegate::sizeHint(const QStyleOptionViewItem &option,
     auto classif = static_cast<const FolderStatusModel *>(index.model())->classify(index);
     if (classif == FolderStatusModel::AddButton) {
         const int margins = aliasFm.height(); // same as 2*aliasMargin of paint
-        QFontMetrics fm(option.font);
+        QFontMetrics fm(qApp->font("QPushButton"));
```

Comment:
Am I blind? Where is that documented?
https://doc.qt.io/qt-5/qguiapplication.html#font

Reply:
Documented there:  https://doc.qt.io/qt-5/qapplication.html#font-2